emgapi/models.py

```diff
@@ -1634,6 +1687,27 @@ def __str__(self):
         return 'Assembly:{} - Sample:{}'.format(self.assembly, self.sample)
 
 
+class MetagenomicsExchangeModel(models.Model):
+    """Model to track Metagenomics Exchange population
+    https://www.ebi.ac.uk/ena/registry/metagenome/api/
+    """
+    last_updated_me = models.DateTimeField(
+        db_column='LAST_UPDATED_ME',
+        auto_now=True
+    )
+    last_populated_me = models.DateTimeField(
+        db_column='LAST_POPULATED_ME',
+        null=True,
+        blank=True,
+        help_text="Date at which this model was last appeared in Metagenomics Exchange"
+    )
+
+    objects_for_population = MetagenomicsExchangeQueryset.as_manager()
+
+    class Meta:
+        abstract = True
```

Comment:
What is this abstract model for? Isn't `last_populated_me` now done by `last_mgx_indexed` in the `MetagenomicsExchangeIndexedModel`? Maybe I misunderstood.

Reply:
good catch *facepalm*, it was left after ebi_search merge by mistake, removed